src/documentation/io/built-in-google-bigquery.md

```diff
@@ -561,9 +568,10 @@ The following example code shows how to apply a `Write` transform to write a
 table if needed; if the table already exists, it will be replaced.
 
 ```java
-{% github_sample /davidcavazos/beam/blob/bigquery-snippets/examples/java8/src/main/java/org/apache/beam/examples/snippets/Snippets.java tag:BigQueryWriteFunction
+{% github_sample /apache/beam/blob/master/examples/java/src/main/java/org/apache/beam/examples/snippets/Snippets.java tag:BigQueryWriteFunction
 %}```
 ```py
+# This Python sample coming soon.
```

Comment:
Found one missing snippet in snippet file  @davidcavazos

Reply:
from my understand, we recommend Write for batch until https://issues.apache.org/jira/browse/BEAM-3289 is finished and it will just do the best thing, so I think we should have some snippet that uses Write even if it's different

Reply:
apache/beam#4771

Reply:
There's no way to provide convert a custom type conversion function to Python (and I'm not sure if we need once since Python writes dictionaries). Following text and the snippet should be dropped from the Python doc.

"The following example code shows how to apply a Write transform to write a PCollection of dictionaries to a BigQuery table. The write operation creates a table if needed; if the table already exists, it will be replaced."